.chloggen/jgm-issue-22850.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/influxdb
```

```suggestion
component: exporter/influxdb
```
Please update PR title and component here to match changes being made.